src/renderer_vk.cpp

```diff
@@ -5665,35 +5665,38 @@ VK_DESTROY
 			, VK_IMAGE_LAYOUT_TRANSFER_SRC_OPTIMAL
 			, _layout
 			, _mip
-			, 1
-			, 0
-			, 1
 			);
```

Comment:
The first `1` (`_levelCount`) should still be there, to only add a layout transition for `_mip`. Otherwise the driver will transition all higher mip levels as well.

Same up top.

Reply:
yep agreed. i changed it back to level=1, baseArrayLayer=1, layerCount = VK_REMAINING_ARRAY_LAYERS for imageMemoryBarrier